src/octoprint/filemanager/analysis.py

```diff
@@ -258,8 +260,9 @@ def throttle():
 
 			self._gcode = gcodeInterpreter.gcode()
 			self._gcode.load(self._current.absolute_path, self._current.printer_profile, throttle=throttle_callback)
-
+						
 			result = dict()
+			result["warning"]=self._gcode.getWarning()
```

Comment:
This is where the min/max coordinates of the object should instead be transferred :)